news/2025-02-13-conda-forge-v1-recipe-support.md

```diff
@@ -0,0 +1,84 @@
+
+## rattler-build and the v1 spec
+
+The v1 recipe format has a number of benefits:
+
+- It always parses as valid YAML which makes it much easier to modify it with the bot infrastructure of conda-forge. While meta.yaml looks like YAML, it can contain Jinja logic incompatible with the YAML specification, which make significantly complicates parsing and automated manipulation. Additionally, the recipe has a published JSON schema which means that the editing experience in VSCode is greatly improved with contextual help.
+- The new recipe executes much faster, which is partially due to some design decisions (no more recursive evaluation) as well as due to rattler-build building on top of a modern re-implementation of the conda standards in Rust.
+- The log output of rattler-build is greatly improved, always showing the user what the final files in the package are and the final metadata.
+- Some features of conda-build, such as multiple outputs, had a lot of implicit behavior. We are fixing that in the v1 recipe.
+
+You can read much more about the v1 recipe format in the [https://rattler.build](https://rattler.build) docs and [CEPs](https://github.com/conda/ceps/).
+
+A simple v1 recipe looks something like the following:
+
+```yaml
+context:
+  # we define named variables in the context instead of `{$ set ... %}` directives
+  version: "23.0.0"
+ 
+package:
+  name: "boltons"
+  # note that we use "GitHub" inspired syntax to access context / Jinja variables
+  version: ${{ version }}
+ 
+source:
+  url: https://github.com/mahmoud/boltons/archive/refs/tags/${{ version }}.tar.gz
+  sha256: 9b2998cd9525ed472079c7dd90fbd216a887202e8729d5969d4f33878f0ff668
+ 
+build:
+  noarch: python
+  script:
+    - python -m pip install . --no-deps -vv
+ 
+requirements:
+  host:
+    - python
+    - pip
+    - setuptools
+  run:
+    - pip
+ 
+about:
+  license: BSD-3-Clause
+  license_file: LICENSE
+```
+
+## How to use the v1 recipe format on conda-forge
+
+If you are adding a new recipe on staged-recipes, then it's easy: just submit a `recipe.yaml` file instead of a `meta.yaml` file.
+In case you already maintain a feedstock, then the conversion can be semi-automated with a tool created by [Hadrien Mary](https://github.com/hadim) called ["feedrattler"](https://github.com/hadim/feedrattler). The tool will take care of the basic conversions steps and uses the `conda-recipe-manager` by Anaconda / Schuyler Martin under the hood to parse the recipe and convert it to the new format.
```

Comment:
```suggestion
In case you already maintain a feedstock, then the conversion can be semi-automated with a tool created by [Hadrien Mary](https://github.com/hadim) called [`feedrattler`](https://github.com/hadim/feedrattler). The tool will take care of the basic conversions steps and uses [`conda-recipe-manager`](https://github.com/conda-incubator/conda-recipe-manager) by Anaconda / [Schuyler Martin](https://github.com/schuylermartin45) under the hood to parse the recipe and convert it to the new format.
```